web/test-data/custom_speeds/baseline_road_zero_speed_one_direction.csv

```diff
@@ -0,0 +1,2 @@
+osm_way_id,max_speed_kph,bwd
+76254223,0,False
```

Comment:
general comment - how can scenario operators figure out which direction is forwards or backwards for a given way when configuring directional speeds? maybe you could manually look up the corresponding stable edge ids in the street export, then check their direction in studio. but i'm not sure we could automate this approach when adding directional speed support to scenario inputs ui

Reply:
Won't respond fully here because we chatted about this last week, but I left some additional thoughts [here](https://github.com/replicahq/graphhopper/pull/171#discussion_r1821551912) after implementing the new validation in our street export code